src/content/other/wind.mdx

```diff
@@ -1,10 +1,13 @@
 ---
 title: Wind Power
-description: While America is lagging in wind power, we can opine to our elected officials
-sectionHeader: Wind Turbines—Capturing Both Our Imaginations and Energy!
+description: Wind is good for more than just cooling you off on a hot summer day
```

Comment:
Feels a little weird to me seeing as a lot of electricity goes to AC.

Reply:
Also, the generated code for this change wasn't committed, so thats why CI's failing, @MattsAttack.

Reply:
by generated do you mean the js files? if so i can go commit those real quick. It might get wonky though

Reply:
We can try in do this in homeroom tomorrow. Since I committed some of the stuff on my pc and some it on my mac book the js files may not be 100% updated on both. also it wont let me look at the js files on my pc

Reply:
Not the js from the mdx, but but js for categories, etc.
The pages are gitignored.